Build fail

OS:Linux Mlint 18

```
$ make
CMake Error at CMakeLists.txt:20 (find_package):
  By not providing "FindPulseAudio.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "PulseAudio", but CMake did not find one.

  Could not find a package configuration file provided by "PulseAudio" with
  any of the following names:

    PulseAudioConfig.cmake
    pulseaudio-config.cmake

  Add the installation prefix of "PulseAudio" to CMAKE_PREFIX_PATH or set
  "PulseAudio_DIR" to a directory containing one of the above files.  If
  "PulseAudio" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/itgoyo/downloads/ncpamixer-1.3.3/build/CMakeFiles/CMakeOutput.log".
Makefile:34: recipe for target 'build/Makefile' failed
make[1]: *** [build/Makefile] Error 1
Makefile:53: recipe for target 'release' failed
make: *** [release] Error 2

```

Hi，there,I google finds nothing to solve this situation.

I had install PulseAudio.

Sorry,My english is poor,hope you can understand what I say.

--- a/README.md
+++ b/README.md
@@ -68,6 +68,8 @@ You find them in ncpamixer.conf
 * CMake
 * C++14 compatible compiler
 
+On Debian(-based) systems, you'd need `libncurses-dev` and `libpulse-dev`.
+
 ### Install
 
 ##### Arch Linux

